myTrip/photo/views.py

```diff
@@ -1,38 +1,56 @@
 """Module generate view for photo requests."""
 
 import json
+
 from django.views.generic.base import View
 from django.http import JsonResponse, HttpResponse
+
+from checkpoint.models import Checkpoint
+from registration.models import CustomUser
+from trip.models import Trip
 from .models import Photo
+from datetime import datetime
```

Comment:
you should add it above, where you import modules from standart django\python packages